Pod/Classes/GDGEntityManager.h

```diff
@@ -55,4 +56,6 @@
 
 - (GDGColumn *)objectForKeyedSubscript:(NSString *)idx;
 
+- (GDGRelation *)relationNamed:(NSString *)relationName;
```

Comment:
The `objectForKeyedSubscript:` could return a `GDGColumn` or a `GDGRelation` depending on the name you have forwarded as a argument :sparkles: :sparkling_heart: